dev/NavigationView/NavigationView.xaml

```diff
@@ -506,10 +521,10 @@
                                         <!-- "Non header" content -->
                                         <Grid x:Name="ItemsContainerGrid" Grid.Row="6" Margin="0,0,0,8">
                                             <Grid.RowDefinitions>
-                                                <RowDefinition Height="Auto" MinHeight="{ThemeResource NavigationViewItemOnLeftMinHeight}"/> <!-- MenuItems -->
-                                                <RowDefinition Height="*" MinHeight="21"/> <!-- Fill spacing -->
+                                                <RowDefinition x:Name="MenuItemsRow" Height="Auto"/> <!-- MenuItems -->
```

Comment:
>MenuItemsRow [](start = 71, length = 12)

Why is this named?

Reply:
Because we forgot to remove that name, fixed now.